client/me/concierge/calendar-card.js

```diff
@@ -32,6 +32,7 @@ class CalendarCard extends Component {
 		disabled: PropTypes.bool.isRequired,
 		isDefaultLocale: PropTypes.bool.isRequired,
 		onSubmit: PropTypes.func.isRequired,
+		signupForm: PropTypes.object.isRequired,
```

Comment:
Since we actually require the timezone for this to work we should require it.

```
    signupForm: PropTypes.shape( {
        timezone: PropTypes.string.isRequired,
    } ).isRequired,
```

Reply:
We could simplify this by just passing in `timezone` instead

Reply:
Right, updated to pass just `timezone`.